### DISCUSSION TITLE
Dynamic Timelapse API

### DISCUSSION THREAD

**Discussion by austindetzel** [blakeblackshear/frigate#10652] [Ideas]
Greetings! Presently, the time-lapse setup offers two options: real-time and 25x speed (which can still be adjusted to a single setting in the config). It would greatly enhance user experience if in the API we could introduce a parameter allowing users to specify the desired processing rate for the time-lapse or request a specific duration (e.g., a 15-second time-lapse capturing the day) dynamically when exporting.

**Comment by NickM-27** [comment]
The config offers a field to adjust the time lapse parameters per camera. The reason this hasn't been implemented is because the next version of frigate will have a fully rewritten ui which includes the ability to easily and quickly scrub through footage for reviewing purposes, filling the need that many currently use time lapses to fill
